### DISCUSSION TITLE
https://xon.sh/ dark mode

### DISCUSSION THREAD

**Discussion by kouhe3** [xonsh/xonsh#5101] [Ideas]
I want dark mode web page. XD

**Comment by gforsyth** [comment]
![2023-03-22_11-46](https://user-images.githubusercontent.com/3596999/226960366-380828fe-34c1-4aba-87b4-7b5b676c804c.png)

Default follows browser theme, but you can click the button to toggle

**Reply by kouhe3** [reply]
> ![2023-03-22_11-46](https://user-images.githubusercontent.com/3596999/226960366-380828fe-34c1-4aba-87b4-7b5b676c804c.png)
> 
> Default follows browser theme, but you can click the button to toggle

No. not doc. I mean web site HOME page.
